ui/decredmaterial/editor.go

```diff
@@ -43,10 +44,12 @@ type Editor struct {
 	Bordered bool
 	//IsPassword if true, displays the show and hide button.
```

Comment:
All comments should start with a space after the slash, and should use the correct capitalization, here the 'i' in isPassword should be small letters, apply these changes for all other comments
```suggestion
	// isPassword if true, displays the show and hide button.
```